app/controllers/accessories_controller.rb

```diff
@@ -8,12 +8,15 @@ class AccessoriesController < ApplicationController
   before_action :authorize_order_detail
   before_action :load_product
 
+  customer_tab :all
+  layout -> { modal? ? false : "application" }
```

Comment:
What I did for this feature was to fix the accessories "modal" to work outside of a modal--which it mostly did except the JS and some styling. After fixing the reservation timestamps, we then redirect here.

I wanted to put the popup on the problem reservation edit page, but in order for the manual and auto scaled accessories to have the right defaults (matching the duration of the reservation) the reservation changes need to be persisted first.

I also tried some stuff with auto-popup of the modal on the receipt page, but that started to get hacky.

The main reason we've used popup modals in the past is to use the same feature, but to retain the current context. At the end of this flow, you're back on the my reservations page, which I think is fine.